{
  "Entries": [
    {
      "RequestUri": "https://api.loganalytics.io/v1/workspaces/32d1e136-gg81-4b0a-b647-260cdc471f68/query",
      "RequestMethod": "POST",
      "RequestHeaders": {
        "Accept": "application/json",
        "Accept-Encoding": "gzip",
        "Authorization": "Sanitized",
        "Content-Length": "29",
        "Content-Type": "application/json",
        "User-Agent": "azsdk-go-azquery/v0.1.0 (go1.19; Windows_NT)"
      },
      "RequestBody": {
        "query": "not a valid query"
      },
      "StatusCode": 400,
      "ResponseHeaders": {
        "Access-Control-Allow-Origin": "*",
        "Access-Control-Expose-Headers": "Retry-After,Age,WWW-Authenticate,x-resource-identities,x-ms-status-location",
        "Connection": "keep-alive",
        "Content-Length": "355",
        "Content-Type": "application/json; charset=utf-8",
        "Date": "Thu, 08 Sep 2022 00:01:31 GMT",
        "Strict-Transport-Security": "max-age=15724800; includeSubDomains",
        "Vary": "Accept-Encoding",
        "Via": "1.1 draft-oms-6b467c4794-wz4rv",
        "X-Content-Type-Options": "nosniff"
      },
      "ResponseBody": {
        "error": {
          "message": "The request had some invalid properties",
          "code": "BadArgumentError",
          "correlationId": "965eca0d-28ec-4c61-b6ec-ed57b77af4ae",
          "innererror": {
            "code": "SyntaxError",
            "message": "A recognition error occurred in the query.",
            "innererror": {
              "code": "SYN0002",
              "message": "Query could not be parsed at \u0027a\u0027 on line [1,4]",
              "line": 1,
              "pos": 4,
              "token": "a"
            }
          }
        }
      }
    }
  ],
  "Variables": {}
}
